p4-16/psa/examples/psa-example-digest.p4

```diff
@@ -170,14 +220,15 @@ control IngressDeparserImpl(packet_out packet,
     CommonDeparserImpl() common_deparser;
```

Comment:
```
psa-example-digest.p4(219): error: digest: Type parameters must be supplied for constructor
    Digest(0) digest;
              ^^^^^^
psa-example-digest.p4(223): error: digest.pack: Cannot unify struct mac_learn_digest_t to T
            digest.pack(meta.mac_learn_digest);
```


Should be `Digest<mac_learn_digest_t>(0) digest`.

Reply:
I've created an issue to remind us about the editing still needed for Digest extern: https://github.com/p4lang/p4-spec/issues/530

Feel free to claim it if you have time.